pkgs/development/tools/fortran-language-server/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, fetchPypi, buildPythonApplication }:
+
+buildPythonApplication rec {
+  pname = "fortran-language-server";
+  version = "1.12.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "7Dkh7yPX4rULkzfJFxg47YxrCaxuHk+k3TOINHS9T5A=";
+  };
+
```

Comment:
Please add a checkPhase and add a pythonImportsCheck

Reply:
This is a pure executable Python application. There is no documented module one could import and there is also no test case, the fls isn't meant to be used interactively from a command line. Which kind of test would you suggest? I am a little bit lost there.

Reply:
You should be able to import it https://github.com/hansec/fortran-language-server/blob/master/fortls.py#L7

Does the cli have a --help or something?

Reply:
Ah nice, I've added the import check and a `fortls --help` invocation and both work fine. Thanks for the ideas :)